src/main/resources/io/jenkins/plugins/restlistparam/RestListParameterDefinition/index.jelly

```diff
@@ -20,7 +20,7 @@
       <select id="${id}-select" class="setting-input" name="value">
         <j:forEach var="value" items="${it.values}" varStatus="loop">
           <f:option value="${value}" selected="${value.equals(it.defaultValue)}">
-            ${value}
+            ${value.displayValue}
```

Comment:
while something fancy like [select element templating](https://select2.org/dropdown#templating) is a option with Select2 I guess this works for now